nucliadb/src/nucliadb/ingest/orm/brain_v2.py

```diff
@@ -216,14 +216,15 @@ def generate_paragraphs(
         user_field_metadata: Optional[UserFieldMetadata],
         replace_field: bool,
         skip_index: Optional[bool],
+        skip_texts_index: Optional[bool],
```

Comment:
Could you rename the one above, so we know which index are we skipping with each param?